strings/ion_laws.json

```diff
@@ -766,7 +766,7 @@
         "THIRTEEN SEQUELS",
         "THREE WISHES",
         "A SITCOM",
-        "THAT GRIEFING FAGGOT GEORGE MELONS",
+        "THAT GRIEFING meatball GEORGE MELONS",
```

Comment:
Atleast make the meatball part fullcaps as well to stick with the rest of the text, same for `death_commando.txt`

Reply:
This is intentional. The sudden all lowercase on a singular word makes it obvious it once was something else. Perhaps if they look at the blame, they can find what it was, making it not only funny in game, but out of game as well.

Reply:
Hehe. 

Meatball.